client/pages/event.jsx

```diff
@@ -1,22 +1,52 @@
-
-import React, { useEffect, useState } from 'react';
+import React, { useMemo, useEffect, useState } from 'react';
 import { useParams } from "react-router-dom";
 import formatDate from '../lib/format-date';
+import GuestList from '../components/guest-list';
+import { GoogleMap, useLoadScript, Marker } from '@react-google-maps/api';
+import usePlacesAutocomplete, { getGeocode, getLatLng } from 'use-places-autocomplete';
+import { Combobox, ComboboxInput, ComboboxPopover, ComboboxList, ComboboxOption } from '@reach/combobox';
 
-export default function Event( ) {
+export default function Event() {
   const [event, setEvent] = useState();
+  const [selected, setSelected] = useState(null);
   const { eventId } = useParams();
+  const libraries = useMemo(() => ['places'], []);
+
+  const mapContainerStyle = {
+    width: '95%',
+    margin: '.5rem',
+    height: '300px',
+    borderRadius: '.25rem'
+  };
+  const { isLoaded } = useLoadScript({
+    googleMapsApiKey: 'AIzaSyAdLGV4RzzLD1SC8fVAshEm_92pcAUgg8s',
+    libraries
+  });
 
   useEffect(() => {
-    console.log('line:9 eventId::: ', eventId);
     fetch(`/api/events/${eventId}`)
       .then((res) => res.json())
       .then((event) => {
         setEvent(event)
-        console.log('line:10 event::: ', event);
       })
-  }, []);
+  }, [eventId]);
+
+  useEffect(() => {
+
+    const displayLocation = async address => {
```

Comment:
I wonder if you had used `function` syntax instead of `arrow function` syntax if the fact it was not being called would have been more clear?
```
useEffect(() => {
  function display(...) {...}
}, []);
```

To me, that is more clear that `display` is never called than the equivalent `arrow function` syntax. This is one reason I prefer to use `function` syntax except when passing short callback functions.